stdlib/build.rs

```diff
@@ -113,7 +113,7 @@ fn write_asm_rs(modules: &ModuleMap) -> Result<()> {
     for (ns, source) in modules {
         let separator_suffix = (0..(89 - ns.len())).map(|_| "-").collect::<String>();
         writeln!(asm_file, "// ----- {ns} {separator_suffix}")?;
-        writeln!(asm_file, "(\"{ns}\", \"{source}\"),")?;
+        writeln!(asm_file, "(\"{ns}\", r##\"{source}\"##),")?;
```

Comment:
I'm not too familiar with this syntax, but could we not simplify it as something like this:
```Rust
writeln!(asm_file, "(\"{ns}\", r#"{source}"#),")?;
```

Reply:
It would also work, but I wasn't sure that there wouldn't be a `"#` combination of characters in the comments. So I added another number sign to be extra safe.

Reply:
So, with single `#` something like this would fail:
```
abc "# def
```
But with `##` this would work. Though, something like this would still fail:
```
abc "## def
```
Is there a way to ensure that all strings would pass?

Reply:
This is the best solution I've been able to find. I haven't seen a way to pass any string

Reply:
I think this is fine for now. We might change the general approach to this as we move to serializing ASTs.